setup.py

```diff
@@ -62,6 +62,10 @@ def run(self):
 	project_urls={
 		'Source': 'https://github.com/TeskaLabs/asab'
 	},
+	install_requires=[
+		'aiohttp>=3.5.4,<4.0',
+		'fastjsonschema>=2.14.1,<3.0',
```

Comment:
Cool. There is an issue #77 that will be resolved by this.

Reply:
Yep, nice. :)